include/deal.II/base/symmetric_tensor.h

```diff
@@ -31,15 +31,15 @@ DEAL_II_NAMESPACE_OPEN
 template <int rank, int dim, typename Number = double>
 class SymmetricTensor;
 
-template <int dim, typename Number>
+template <int dim, typename Number = double>
```

Comment:
I deleted the implementation specialization for `Number = double` but instead I added this default template type argument here.